### PR TITLE
add subheadings for workspaces

### PR DESCRIPTION
**Pull request by btwisted**

### CODE REVIEW

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+[![API lifecycle](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-publish-or-removepWS.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-publish-or-removepWS.png)
+
+
+<h3 id="delete">Deleting a workspace</h3>
+
+
+[![delete workspace](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-delete_WS.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-delete_WS.png)
+
+
+<h3 id="join">Joining Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: @btwisted  can you replace this with markdown? I think @joyous-joyce like to have all new post written in markdown to keep it consistent.

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+A user can join a workspace to work on collections, environments, monitors, mocks and integrations in it.
+
+[![join ws menu](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-join-WS-mainScreen-menu.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-join-WS-mainScreen-menu.png)
+
+[![join two](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-join-second-step.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-join-second-step.png)
+
+[![join three](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-join-third-step1.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-join-third-step1.png)
+
+
+<h3 id="share">Sharing Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+#### Sharing personal Workspaces
+
+[![](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-elipsis-menu-personal.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-elipsis-menu-personal.png)
+
+[![share personal](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-share-personal-WS.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-share-personal-WS.png)
+
+[![blank]()]()
+
+<h3 id="shareCollection">Sharing collections in Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+[![share get link](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-getLink-collections.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-getLink-collections.png)
+
+
+#### Sharing collections in the sidebar
+
+[![share sidebar](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-share-collection-sidebar.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-share-collection-sidebar.png)
+
+Sharing in a workspace will be the same action as adding from a private workspace to a team, public or enterprise workspace. However, when an item is shared with a team, it will have an additional owner (the team). If an item is removed from the default workspace of a user, it will still remain in the team's default workspace and will have to be explicitly removed from the team by the team admin or someone with edit access.
+
+<h3 id="view">Viewing Details</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+[![details personal](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-view-details-personal.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-view-details-personal.png)
+
+#### Viewing details for team Workspaces 
+
+[![API lifecycle](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-elipsis-menu-team.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-elipsis-menu-team.png)
+
+[![details team](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-team-view-details.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-team-view-details.png)
+
+<h3 id="rename">Renaming Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+#### Renaming personal Workspaces 
+
+[![rename personal](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-edit-workspace-details-personal.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-edit-workspace-details-personal.png)
+
+#### Renaming team Workspaces 
+
+[![rename team](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-edit-WS-details-team.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-edit-WS-details-team.png)
+
+<h3 id="shareElements">Sharing elements to Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+In this example, we share a collection to a Workspace.
+
+[![share elements](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-share-elements-dashboard.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-share-elements-dashboard.png)
+
+[![share elements-collections](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-share-elements-collection.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-share-elements-collection.png)
+
+<h3 id="pub">Publishing Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+
+[![API lifecycle](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-publish-or-removepWS.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-publish-or-removepWS.png)
+
+<h3 id="leave">Leaving Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+[![leave workspaces](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-leave-WS-team.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-leave-WS-team.png)
+
+<h3 id="edit">Editing descriptions in Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+
+[![edit descr]()]()
+
+
+<h3 id="manageTeam">Managing team members in Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-02-14-managing_workspaces.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+page_id: "managing_workspaces"
+warning: false
+---
+
+                                          **THIS IS A WORKING DRAFT-NOT READY FOR REVIEW**
+
+
+* [Adding Workspaces](#add)
+
+<h3 id="add">Adding Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-02-14-managing_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+
+[![remove sidebar](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-remove-WS-via-sidebar.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-remove-WS-via-sidebar.png)
+
+
+#### Removing Workspaces in the dashboard
+
+[![API lifecycle](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-publish-or-removepWS.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-publish-or-removepWS.png)
+
+
+<h3 id="delete">Deleting a workspace</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md [hunk 13]
```diff
@@ -0,0 +1,148 @@
...
+
+* [Renaming Workspaces](#rename)
+
+* [Sharing elements to Workspaces](#shareElements)
+
+* [Publishing Workspaces](#pub)
+
+
+
+<h3 id="remove">Removing Workspaces</h3>
```
Comment (66eli77) [inline _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: Replace with markdown

File: _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md (hunk 12 above)
Comment (btwisted) [reply _posts/01_postman/12_workspaces/2018-01-22-using_workspaces.md]: try it now

**Pull request merged**